src/test/java/com/redhat/exhort/integration/backend/BackendUtilsTest.java

```diff
@@ -34,10 +34,16 @@
 class BackendUtilsTest {
 
   private BackendUtils backendUtils;
+  private BackendUtils backendUtilsFakeTimer;
+  private int numberOfRequests;
```

Comment:
set it as a constant and initialize it here

Reply:
done.